File: StripeGate/src/main/java/com/androidafe/stripegate/models/PaymentResponse.java
package com.androidafe.stripegate.models;

public class PaymentResponse {
    private String id;
    private String client_secret;

    public PaymentResponse(String id, String client_secret) {
        this.id = id;
        this.client_secret = client_secret;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getClientSecret() {
        return client_secret;
    }

    public void setClientSecret(String client_secret) {
        this.client_secret = client_secret;
    }
}

File: StripeGate/src/main/java/com/rahad/stripegate/models/PaymentResponse.kt
package com.rahad.stripegate.models

data class PaymentResponse(
    val id:String,
    val `object`:String,
    val amount:String,
    val amount_received:String,
    val application:String,
    val cancellation_reason:String,
    val capture_method:String,
    val client_secret:String,
    val confirmation_method:String,
    val customer:String,
)